source/Hardpoint.h

```diff
@@ -95,6 +100,8 @@ class Hardpoint {
 	bool isTurret = false;
 	bool isFiring = false;
 	bool wasFiring = false;
+	// Indicates if this hardpoint disallows converging (guns only).
+	bool isParallel = false;
```

Comment:
Let's move this (and `isTurret`) up in order here, so that the members we are initializing in the constructor are adjacent in the class's memory layout. (This will also creates two logical groups of hardpoint fields - the static as-initialized ones, and the dynamic ones that are frequently modified.)